pancancer_utilities/scripts/classify_cancer_type.py

```diff
@@ -18,6 +18,7 @@
     process_y_matrix,
     process_y_matrix_cancertype,
```

Comment:
`process_y_matrix_cancertype` is unused in this file, as well as `import pickle as pkl`

Reply:
Good catch, I copied those over from another script but ended up not using them.